deploy/clowdapp.yml

```
@@ -1381,6 +1387,8 @@ parameters:
   value: host-inventory
 - name: PATH_PREFIX
   value: api
+- name: CONSOLEDOT_HOSTNAME
+  value: 'https://console.redhat.com'
```

It only contains the hostname, not the protocol
```suggestion
  value: 'console.redhat.com'
```